### ISSUE TITLE
[FR]: Add option to fully close the app

### ISSUE THREAD

**Issue by Build-0-Matic** [martinrotter/rssguard#906]
### Brief description of the feature request

It would be nice to have the option to fully close the app when the X button is clicked.

**Comment by martinrotter** [comment]
You mean when you have tray icon ENABLED?

**Comment by Build-0-Matic** [comment]
I hadn't found that option... I disabled it and now it works fine.
But a minimize to tray option instead of close to tray would be a great addition.

**Comment by Build-0-Matic** [comment]
Yeah, I just found it...
I just figured out something: all the functions I need are probably present, but the settings are not spread out like I would have thought.
Like the General tab has only two functions, that I feel should be combined with the Tray Area functions since they affect the entire program.
And there are some synonyms that are used that are not common. Like "Hide main window when minimized" is more commonly referred to as "Minimize to tray" and so on. When you navigate the interface without stopping and reading (like a lot of users do), you tend to miss those options and features.

But apart from that, the program is pretty darn good.
